### DISCUSSION TITLE
Vou ganhar um macbook pro i7, vendo ou uso ?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1350] [Discussão]
Um cliente meu me deve a um tempo, como trabalha com importação ele me pagou com um macbook pro i7 16GB 512GB SSD, sempre usei linux e pc, então ainda estou na dúvida se fico de vez com o mac e migro ou vendo...

vou ter realmente muita vantagem com um macpro? meu trabalho é focado em react, react native.

**Comment by bizarro** [comment]
Como você trabalha com React Native, acho que seria uma ótima ideia começar a utilizar ele para desenvolver, pois você vai poder testar e gerar a versão iOS dos aplicativos com muito mais facilidade.

A diferença entre o Linux não é muito grande também, então acho que você vai se adaptar bem mais rápido do que imagina.

**Comment by Juuhz** [comment]
Cara,

Comprei meu mac #1329, e foi bem positivo, me da liberdade por conta da durabilidade da bateria e portabilidade. Se você é uma pessoa que precisa sempre estar conectado, ter um mac é uma boa pedida. 

O meu é inferior ao seu e voa, não trava, responde mega rápido e os atalhos do keyboard / trackpad te dão bastante autonomia. 

Recomendo você ficar com ele e testar, até porquê mac é algo que não desvaloriza. O meu é de 2015 e paguei 4,5 nele. 

Abraços.

**Comment by romulosardinha** [comment]
@fabianoschaefer ai fico rico depois dos investimentos e viro empreendedor de palco ! kkk

**Comment by romulosardinha** [comment]
@lhbzr to pensando nisso mesmo, esperando não levar um tranco grande com a mudança

**Comment by romulosardinha** [comment]
@Juuhz hum... boa ideia essa ai de revenda né. ele não desvaloriza rápido.

**Comment by israelcena** [comment]
@rmlSardinha, Amigo,
Imagina que você é piloto profissional, e um cliente que lhe deve lhe da uma Ferrari como forma de pagamento. 
Você já tem um "Prisma" pra correr, mas agora tem uma Ferrari. 
Você vende a Ferrari porque você já corre super bem com o Prisma ou usa a Ferrari para correr mais e competir ainda melhor nas corridas ?

**Comment by romulosardinha** [comment]
@israelcena  kkkk adorei a comparação, se chamar o team pc de usuários de prismas vai gerar hate kk

**Comment by vNNi** [comment]
Nunca tinha usado um mac, até ano passado... agora falo com muita certeza que é algo muito difícil de ficar sem e o mais importante, não é difícil de migrar!

**Comment by romulosardinha** [comment]
@vNNi legal, esse tipo de experiência de causa que estou procurando, bom ... ele deve chegar daqui a duas semanas, vamos ver se vicio mesmo

**Comment by fakenickels** [comment]
Você vai fazer muito mais grana com o Mac do que a grana de vender ele, vai por mim. Melhor investimento que já fiz, tenho um Macbook Pro com as mesmas specs que as suas.

**Comment by romulosardinha** [comment]
@grsabreu , esse hardware no mac rende mais do que o mesmo hardware em um pc ? ouvi falar isso, é verdade ?

**Comment by fakenickels** [comment]
é verdade @rmlSardinha , o gerenciamento de recursos e o próprio OS e tooling de dev para o Mac é infinitamente superior ao do Linux e Windows. As coisas apenas funcionam de primeira.
Eu não acreditava nisso tb até usar um Mac.

**Comment by israelcena** [comment]
Pessoal que critica os "produtos mac", em sua maioria, se dividem em dois grupos:

1. Os que nunca usaram 
2. Os que não criaram coragem ainda para investir em um (Dizem também que "não tem dinheiro").

**Comment by nandomoreirame** [comment]
Ótimo argumento @israelcena vou usar sempre kkkk

**Comment by onlurking** [comment]
Mac é um `BSD` de pelúcia, é unix-like o suficiente pra ter uma tooling de desenvolvimento absurda(capaz de competir com distros Linux de certa forma), e ao mesmo tempo é relevante o suficiente no mercado pra ter aplicações de escritório e design, como Office, Suite Adobe e o [Sketch](https://www.sketchapp.com).

Fora aplicações excelentes como [Things](https://culturedcode.com/things), [CodeKit](https://codekitapp.com/) e o [Alfred](https://www.alfredapp.com) com suas [integrações](https://github.com/zenorocha/alfred-workflows#alfred-workflows).

Ele tem o melhor dos dois mundos, entretanto eu particularmente nunca usaria um Mac por causa do sistema ser extremamente fechado e possuir uma assistência muito cara no Brasil(e sou meio paranóico com esse papo de [obsolescência programada](https://www.google.com/search?client=firefox-b-ab&q=apple+planned+obsolescence+lawsuit)).

É o que a galera falou aí acima, use ele por um tempo, se lhe atender melhor que teu notebook atual(com Linux) fique com ele.

**Comment by romulosardinha** [comment]
@israelcena kkkk rapaz, o que tem de gente q me cerca que odeia mac kk, se eu falar q é pq nunca usaram vão me crucificar kk

**Comment by romulosardinha** [comment]
@onlurking  pois é , essa semana isso será definido finalmente. se ocorrer tudo como quero posto a foto do setup
